packages/core/src/core/subagent.test.ts

```diff
@@ -767,7 +769,7 @@ describe('subagent.ts', () => {
         // Use fake timers to reliably test timeouts
         vi.useFakeTimers();
 
-        const { config } = await createMockConfig();
+        const { config } = await createMockConfig({}, { useRipgrep: false });
```

Comment:
I'm not 100% sure why, but for some reason I had to disable ripgrep here and in the below test to get the subagent tests to pass on windows, otherwise they were mysteriously timing out. I can investigate more if desired, but it may just be a timers issue.